spec/decorators/zine_decorator_spec.rb

```diff
@@ -9,8 +9,23 @@
     described_class.new(zine)
   end
 
-  describe '#cover_image_tag' do
+  describe '#id' do
+    subject do
+      described_class.new(build :zine, id: 1234)
+    end
+    it 'returns the zine id' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.